keps/sig-cluster-lifecycle/kubeadm/4656-add-kubelet-instance-configuration/kep.yaml

```diff
@@ -0,0 +1,26 @@
+title: Add kubelet instance configuration to configure CRI socket for each node
+kep-number: 4656
+authors:
+  - "@HirazawaUi"
+owning-sig: sig-cluster-lifecycle
+status: provisional
+creation-date: 2024-05-23
+reviewers:
+  - "@neolit123"
+  - "@pacoxu"
+approvers:
+  - "@neolit123"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: implementable
```

Comment:
we should leave this at `stage; alpha`
it doesn't matter for kubeadm keps and PRR